### PR TITLE
MHR search result checkbox UX fix

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12802

*Description of changes:*
Lien information state is kept when going back from summary page.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue [hunk 1]
```diff
@@ -336,9 +336,10 @@ export default defineComponent({
       localState.searched = true
       localState.searchType = getSearchedType.value?.searchTypeUI || ''
       localState.results = sortByStatus(localState.activeResults)
-      if (!props.isReviewMode) {
-        localState.results = localState.results.map(result => ({ ...result, includeLienInfo: false }))
-      }
+      localState.results = localState.results.map(result => {
+        // includeLienInfo needs to be initialized or something weird will happen
+        return result.includeLienInfo !== true ? { ...result, includeLienInfo: false } : result
+      })
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/mhr/SearchedResults.vue:342]: So you want to do this for both Results selection and Results selection review modes?

Comment (garyfan1) [reply ppr-ui/src/components/tables/mhr/SearchedResults.vue:342]: it doesn't really make any difference in terms of UX behavior, just... less code.

Comment (cameron-eyds) [reply ppr-ui/src/components/tables/mhr/SearchedResults.vue:342]: Ok sounds good, if it was intentional!

Resolved

**Review by cameron-eyds — APPROVED** [review]

**Review by dimak1 — APPROVED** [review]

**Review by ChasBoucher — COMMENTED** [review]
Looks Good

**Review by ChasBoucher — COMMENTED** [review]
Looks Good

**Pull request merged**